src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Location/LocationWithNMEA/LocationWithNMEA.xaml.cs

```diff
@@ -81,10 +81,10 @@ private async Task Initialize()
 
         private void UpdateNmeaMessageLabel(object sender, NmeaMessageEventArgs e)
         {
-            Dispatcher.BeginInvoke(delegate ()
+            Dispatcher.BeginInvoke(new Action(() =>
```

Comment:
The `new Action(` isn't needed here; you can provide the anonymous method directly.

Reply:
I get the same build failure with the `() =>` pattern. Keeping this as action.